src/Models/ElementContent.php

```diff
@@ -31,7 +31,8 @@ public function getCMSFields()
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
             $fields
                 ->fieldByName('Root.Main.HTML')
-                ->setTitle(_t(__CLASS__ . '.ContentLabel', 'Content'));
+                ->setTitle(_t(__CLASS__ . '.ContentLabel', 'Content'))
+                ->setSchemaComponent('HtmlEditorField');
```

Comment:
Shouldn't this be the default on HTMLEditorField in core? AFAIK it won't affect anything unless it's being rendered in a React form anyway, and we'd prefer that it works everything rather than just this module

Reply:
Seems valid, since I've added the component to core anyway. This was initial setup that I then forgot about. I'll change it :)